repos/system_upgrade/common/actors/scankernelcmdline/tests/test_scankernelcmdline.py

```diff
@@ -0,0 +1,63 @@
+from leapp.libraries.actor import scankernelcmdline
+from leapp.libraries.common.testutils import CurrentActorMocked, produce_mocked
+from leapp.libraries.stdlib import api, CalledProcessError
+from leapp.models import KernelCmdline, KernelCmdlineArg
+
+
+def mock_cmd_output():
+
+    mocked_cmd = 'BOOT_IMAGE=(hd0,msdos1)/vmlinuz-xxx root=UUID=some_uid ro console=tty0 '\
+        'console=ttyS0,115200 rd_NO_PLYMOUTH biosdevname=0 net.ifnames=0 crashkernel=auto'
```

Comment:
Omit using backslach to join multiple lines at any cost - consider them forbidden in python code in this project. Use rather brackets:
```suggestion
    mocked_cmd = (
        'BOOT_IMAGE=(hd0,msdos1)/vmlinuz-xxx root=UUID=some_uid ro console=tty0 '
        'console=ttyS0,115200 rd_NO_PLYMOUTH biosdevname=0 net.ifnames=0 crashkernel=auto'
    )
```
the same below